### PR TITLE
Added twitter link to readMe

### PR DESCRIPTION
**Pull request by rukundob451**
Twitter link was added to readMe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,3 +14,5 @@ Here are some ideas to get you started:
 - 😄 Pronouns: ...
 - ⚡ Fun fact: ...
 -->
+## How to reach me ##
+[linkedin](https://twitter.com/ascorbic)
```
Comment (ascorbic) [inline README.md]: Not LinkedIn!

Comment (rukundob451) [reply README.md]: Am going to clarify that in a few

**Comment by rukundob451** [comment]
Thanks for correcting me, I'll be really glad for a follow back on here https://github.com/rukundob451 and we can connect on linkedin https://www.linkedin.com/in/benjamin-rukundo-539ab01a6/

**Comment by ascorbic** [comment]
I'm going to close this. It isn't how I'd like to handle this. Thanks anyway.

**Comment by rukundob451** [comment]
Okay, no problem. Have a great time.

**Pull request closed without merging**